Jenkinsfile

```diff
@@ -0,0 +1,184 @@
+#!groovy
+
+def initializeEnvironment() {
+  env.DRIVER_DISPLAY_NAME = 'CassandraⓇ Quarkus Extension'
+
+  env.GIT_SHA = "${env.GIT_COMMIT.take(7)}"
+  env.GITHUB_PROJECT_URL = "https://${GIT_URL.replaceFirst(/(git@|http:\/\/|https:\/\/)/, '').replace(':', '/').replace('.git', '')}"
+  env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
+  env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
+
+  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.6.0"
+  env.PATH = "${env.MAVEN_HOME}/bin:${env.PATH}"
+  env.JAVA_HOME = sh(label: 'Get JAVA_HOME',script: '''#!/bin/bash -le
+    . ${JABBA_SHELL}
+    jabba which ${JABBA_VERSION}''', returnStdout: true).trim()
+  env.JAVA8_HOME = sh(label: 'Get JAVA8_HOME',script: '''#!/bin/bash -le
+    . ${JABBA_SHELL}
+    jabba which 1.8''', returnStdout: true).trim()
+
+
+  sh label: 'Display Java and environment information',script: '''#!/bin/bash -le
+    # Load CCM environment variables
+
+    . ${JABBA_SHELL}
+    jabba use ${JABBA_VERSION}
+
+    java -version
+    mvn -v
+    printenv | sort
+  '''
+}
+
+def buildQuarkusExtension(jabbaVersion) {
+  withEnv(["BUILD_JABBA_VERSION=${jabbaVersion}"]) {
+    sh label: 'Build driver', script: '''#!/bin/bash -le
+      . ${JABBA_SHELL}
+      jabba use ${BUILD_JABBA_VERSION}
+
+      mvn -B -V install -DskipTests -Dmaven.javadoc.skip=true
+    '''
+  }
+}
+
+def executeTests() {
+  sh label: 'Execute tests', script: '''#!/bin/bash -le
+    # Load CCM environment variables
+    
+    . ${JABBA_SHELL}
+    jabba use ${JABBA_VERSION}
+
+    if [ "${JABBA_VERSION}" != "1.8" ]; then
+      SKIP_JAVADOCS=true
+    else
+      SKIP_JAVADOCS=false
+    fi
+
+    printenv | sort
+    
+    mvn -B -V verify -Dmaven.javadoc.skip=${SKIP_JAVADOCS}
+  '''
+}
+
+def executeTestsNative() {
+  sh label: 'Execute tests Native', script: '''#!/bin/bash -le
+    # Load CCM environment variables
+    
+    . ${JABBA_SHELL}
+    jabba use ${JABBA_VERSION}
+    jabba use ${GRAALVM_VERSION}
+
+    if [ "${JABBA_VERSION}" != "1.8" ]; then
+      SKIP_JAVADOCS=true
+    else
+      SKIP_JAVADOCS=false
+    fi
+
+    printenv | sort
+    
+    mvn -B -V verify -Dnative -Dmaven.javadoc.skip=${SKIP_JAVADOCS}
```

Comment:
If I remember correctly, the way I created the `native` profile it will execute both non-native and native tests. So here we are probably executing the native tests twice, and building a native image twice.